src/dmd/escape.d

```diff
@@ -897,7 +909,37 @@ bool checkNewEscape(Scope* sc, Expression e, bool gag)
  */
 bool checkReturnEscape(Scope* sc, Expression e, bool gag)
 {
-    //printf("[%s] checkReturnEscape, e = %s\n", e.loc.toChars(), e.toChars());
+    //printf("[%s] checkReturnEscape, e: %s\n", e.loc.toChars(), e.toChars());
+
+    /* If constructing a literal, i.e.:
+     *   structliteral.ctor(args)
+     * then treat as returning each arg.
+     */
+version (none)
```

Comment:
Why is this `version(none)`? Thats an awful lot of nesting, worth de Morgan'ing?